### DISCUSSION TITLE
Intelligent Tiering

### DISCUSSION THREAD

**Discussion by AmirMohG** [minio/minio#15822] [Q&A] [answered]
Hello there,
is there an easy way to implement intelligent tiering in MinIO (to save you the trouble of looking it up, I'm referring to the ability to move buckets between clusters based on usage rather than transitioning based on time) or do I have to go through the trouble of monitoring the usage on my buckets and move them between clusters manually? I'm also open to any opinions you might have on whether or not I should be implementing this feature.

**Comment by klauspost** [comment, marked as answer]
No. MinIO does no writes on reads. This is important to performance, scalability and reducing needless wear on storage.

Therefore automatic tiering is only based on static properties like creation time, prefixes, tags, etc.

You are welcome to do this yourself, but you will have to track access count + times, through notifications or audit logging.

**Reply by AmirMohG** [reply]
Thanks
